src/web/security/core/src/main/resources/GeoServerApplication_fr.properties

```diff
@@ -20,11 +20,11 @@ security.auth.description=Fournisseurs d'authentification et configuration
 security.passwd.title=Mots de passe
 security.passwd.description=Configuration du mot de passe
 security.userGroupRole.title=Utilisateurs, Groupes et R�les
-security.userGroupRole.description=Gestion des utilisateurs, groupes, et r�les
+security.userGroupRole.description=Gestion des utilisateurs, groupes et r\u00F4les
 security.data.title=S�curit� des donn�es
 security.data.description=�diter les r�gles d'acc�s aux donn�es
-security.service.title=S�curit� du service
-security.service.description=�diter les r�gles d'acc�s au service
+security.service.title=S\u00E9curit\u00E9 des services
+security.service.description=\u00C9diter les r\u00E8gles d'acc\u00E8s aux services
```

Comment:
looks like that is correct encoding for java properties file; what is the editing experience in web late for `É` ?